Gopkg.lock

```diff
@@ -348,7 +350,7 @@
     "leveldb/util",
   ]
   pruneopts = "UT"
-  revision = "6b91fda63f2e36186f1c9d0e48578defb69c5d43"
+  revision = "f9080354173f192dfc8821931eacf9cfd6819253"
```

Comment:
As we're approaching 1.0 release, we should be mindful about upgrading dependencies. This includes listing and auditing new changes and testing them. I've checked leveldb (minor changes).